### ISSUE TITLE
False positive for W0612

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#196]
Originally reported by: **the mulhern (BitBucket: [the_mulhern](http://bitbucket.org/the_mulhern))**

---

Mini-example:

```
#!python

path = None

def junk():
    global path
    from os import path

def junk2():
    print("%s" % path)
```

Reports W0612 (unused-variable) for line "from os import path".

If code is instead:

```
#!python

path = None

def junk():
    global path
    path = "some"

def junk2():
    print("%s" % path)
```

this error is not reported. Clearly some confusion is caused by 'path' being set by means of an import, rather than an assignment.

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/196
